src/main/java/me/xemor/skillslibrary2/effects/LaunchEffect.java

```diff
@@ -1,21 +1,23 @@
 package me.xemor.skillslibrary2.effects;
 
+import me.xemor.configurationdata.entity.EntityData;
 import org.bukkit.Location;
 import org.bukkit.World;
 import org.bukkit.configuration.ConfigurationSection;
 import org.bukkit.entity.Entity;
-import org.bukkit.entity.EntityType;
 import org.bukkit.entity.Player;
 import org.bukkit.util.Vector;
 
 public class LaunchEffect extends Effect implements EntityEffect, TargetEffect {
 
-    private final EntityType entityType;
+    private final EntityData entityData;
     private final double velocity;
 
     public LaunchEffect(int effect, ConfigurationSection configurationSection) {
         super(effect, configurationSection);
-        this.entityType = EntityType.valueOf(configurationSection.getString("entity", "FIREBALL"));
+
```

Comment:
Yeahh, I did mention that I wasn't sure if it was possible to add backwards compat for this, it may be easier for me to implement a method in the EntityData rewrite that accepts just an EntityType to resolve this.

Reply:
Yeah good idea

Reply:
Resolved in https://github.com/Xemorr/SkillsLibrary/pull/4/commits/273e528fcb98c95f7f521bf3fc394f88ce7a4178